doc/configuration.rst

```diff
@@ -64,6 +64,27 @@ port 53867 and use a baud rate of 115200 with the RFC2217 protocol.
 Used by:
   - `SerialDriver`_
 
+ModbusRTU
++++++++++
+Descriped the resource required to use the Modbus RTU driver
```

Comment:
A bit more explanation would be helpful, maybe a link to a Modbus RTU protocol description. Is Modbus RTU always used over serial interfaces?